bin/apostrophe

```diff
@@ -7,46 +7,36 @@ var util = require('../lib/util');
 var fs = require('fs');
 
 program.version(getVersion());
-
 // Get version of apostrophe-cli and apostrophe.
 function getVersion() {
-  var pkginfo = require('pkginfo')(module, 'version');
+  require('pkginfo')(module, 'version');
```

Comment:
Why is this still here? Isn't this what crashes?

Reply:
No, this is looking at the installed CLI module. But it's easy to replace.